WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -0,0 +1,13 @@
+<ul class="list-group">
+  <li class="list-group-item"><strong><%= title %></strong></li>
+  <% competitions.each_with_index do |competition, index| %>
+    <li class="list-group-item <%= competition.is_over? ? "past" : "not-past" %>">
+      <span class="date"><%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %>, <%= competition.year %></span>
+      <span class="competition-info">
+        <p class="competition-link"><%= link_to competition.name, competition_path(competition) %></p>
+        <p class="location"><strong><%= competition.country.name %></strong>, <%= competition.cityName %></p>
+        <%=md(competition.venue, target_blank: true) %>
```

Comment:
This still needs to be changed to use [`processLinks_to_markdown`](https://github.com/jfly/worldcubeassociation.org/blob/port-competition-to-rails/WcaOnRails/db/migrate/20160109070723_convert_process_links_format_to_markdown.rb#L2-L4).